test/data/special_cases/trim_input/trim_input.yml

```diff
@@ -0,0 +1,81 @@
+readme: |
+  This is a README
+  More readme lines
+
+input:
+  - A.gvcf: |
+      ##fileformat=VCFv4.2
```

Comment:
The gVCF header will generally be the same for each sample in the case, except for the very last entry giving the sample identifier. Since we're usually just giving a handful of records per sample, I think it'd be worthwhile to be able to write the header just once in each YAML. When you write out each temporary gVCF file to disk, you could do something like `header+"\t"+sample+"\n"+vcf_body`